api/modules/reports_list.module.php

```diff
@@ -10,11 +10,11 @@ class ApiModuleReportsList extends ApiModule
 {
     public function content()
     {
-        global $statuses, $mysql;
+        global $mysql;
 
         $query = "SELECT * FROM `reports` JOIN `vandalism` ON `revertid` = `id`";
         if (isset($_REQUEST['status']) && (!empty($_REQUEST['status']) || $_REQUEST['status'] == '0')) {
-            if (array_key_exists($_REQUEST['status'], $statuses)) {
+            if (array_key_exists($_REQUEST['status'], STATUSES)) {
```

Comment:
Maybe add a helper function `isValidStatus` or such to keep the indexing logic of STATUSES in dbfunctions... this is sort of a redundant check, could just verify it's an `int` as the query will return no rows if it's not a valid status.